examples/example-project/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "example-project",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "start": "ts-node src/index.ts",
```

Comment:
`npm run start` is not working.
It looks like you need to add:
```json
  "type": "module",
```

and modify the `start` script to:
```json
    "start": "node --loader ts-node/esm src/index.ts",
```

Reply:
It works for me, what do you see?

Reply:
switched to tsx instead of ts-node